src/Symfony/Component/Form/FormErrorIterator.php

```diff
@@ -266,6 +267,27 @@ public function seek($position)
     }
 
     /**
+     * Creates iterator for errors with specific codes.
+     *
+     * @param string|string[] $codes The codes to find
+     *
+     * @return static New instance which contains only specific errors.
+     */
+    public function findByCodes($codes)
```

Comment:
I doubt it.. forms can be nested: https://github.com/Koc/symfony/blob/29a3a7e0d6ea12d686789f549afcfb1137f85367/src/Symfony/Component/Form/FormErrorIterator.php#L47

So this should still be fixed.. and yeah.. imo. this cant be tight to `ConstraintViolation::getCode`.